tests/upgrade/postgres_run.sh

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+
+set -euo pipefail
+
+# Tests upgrade to Postgres.
+
+TEST_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
+
+source "$TEST_ROOT/scripts/lib.sh"
+source "$TEST_ROOT/scripts/ci/lib.sh"
+source "$TEST_ROOT/scripts/ci/sensor-wait.sh"
+source "$TEST_ROOT/tests/scripts/setup-certs.sh"
+source "$TEST_ROOT/tests/e2e/lib.sh"
+source "$TEST_ROOT/tests/upgrade/lib.sh"
+
+test_upgrade() {
+    info "Starting Postgres upgrade test"
+
+    if [[ "$#" -ne 1 ]]; then
+        die "missing args. usage: test_upgrade <log-output-dir>"
+    fi
+
+    local log_output_dir="$1"
+
+    require_environment "KUBECONFIG"
+
+    export_test_environment
+
+    REPO_FOR_TIME_TRAVEL="/tmp/rox-postgres-upgrade-test"
+    DEPLOY_DIR="deploy/k8s"
+    QUAY_REPO="rhacs-eng"
+    REGISTRY="quay.io/$QUAY_REPO"
```

Comment:
Do we need to put the is_CI check here like the other version of run has?  I'm not sure these are being executed outside of CI though.

    if is_CI; then
        REGISTRY="quay.io/$QUAY_REPO"
    else
        REGISTRY="stackrox"
    fi

Reply:
We cannot use docker.io any more.